src/docdb/tree/DocDBAccountTreeItemBase.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { DatabaseAccount } from 'azure-arm-cosmosdb/lib/models';
+import { DatabaseAccount } from '@azure/arm-cosmosdb/esm/models';
```

Comment:
Please switch these to use the `Models` object from the base of the npm package, which is more of an npm package's "official" api. Something like this

```
import { ResourceManagementModels } from '@azure/arm-resources';
```

Reply:
Is this a newer preference? This has been how it's used everywhere. 
For eg. [appservice](https://github.com/microsoft/vscode-azureappservice/blob/25cb5fd10fe62d5992626826858bc10d70677904/src/explorer/DeploymentSlotsTreeItem.ts#L7)

Reply:
I mean it's not super new, but yeah it changed sometime after we originally created the extensions. For example the original app service extension referenced v1.0.0-preview of "azure-arm-website" which didn't even have the models object:
<img width="352" alt="Screen Shot 2019-06-21 at 4 08 42 PM" src="https://user-images.githubusercontent.com/11282622/59951637-29ffef00-943f-11e9-91cc-9055cf2e03cc.png">

Now we're on a much later version that does:
<img width="621" alt="Screen Shot 2019-06-21 at 4 09 43 PM" src="https://user-images.githubusercontent.com/11282622/59951651-32f0c080-943f-11e9-84da-ea3bb9180c95.png">

Functions is already moved over: https://github.com/microsoft/vscode-azurefunctions/blob/795ab870b56e133e2c360b3bf08df9419b792d4a/src/tree/FunctionsTreeItem.ts#L6
Docker is moved over: https://github.com/microsoft/vscode-docker/blob/2fae7a525c41f6f8375f51fc6a3598e2d05e5ca4/src/tree/registries/azure/AzureTaskRunTreeItem.ts#L6

There's a tslint rule here that we can eventually turn on: https://palantir.github.io/tslint/rules/no-submodule-imports/

Reply:
Thanks for the heads up. 

I turned on the rule, it looks like there are other places where the rule is broken. Leaving those at the moment.